dist/pom.xml

```diff
@@ -51,25 +51,148 @@
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>set-included-buidlvers</id>
+                                <goals><goal>run</goal></goals>
+                                <phase>generate-sources</phase>
+                                <configuration>
+                                    <target xmlns:ac="antlib:net.sf.antcontrib">
+                                        <exec executable="python" outputproperty="included_buildvers">
+                                            <arg value="${spark.rapids.source.basedir}/build/get_buildvers.py"/>
+                                            <arg value="no_snap_with_databricks"/>
+                                            <arg value="${spark.rapids.project.basedir}/pom.xml"/>
+                                        </exec>
+                                    </target>
+                                    <exportAntProperties>true</exportAntProperties>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>snapshotsWithDatabricks</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>set-included-buidlvers</id>
+                                <goals><goal>run</goal></goals>
+                                <phase>generate-sources</phase>
+                                <configuration>
+                                    <target xmlns:ac="antlib:net.sf.antcontrib">
+                                        <exec executable="python" outputproperty="included_buildvers">
+                                            <arg value="${spark.rapids.source.basedir}/build/get_buildvers.py"/>
+                                            <arg value="snap_and_no_snap_with_databricks"/>
+                                            <arg value="${spark.rapids.project.basedir}/pom.xml"/>
+                                        </exec>
+                                    </target>
+                                    <exportAntProperties>true</exportAntProperties>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
```

Comment:
There is a lot of copy-and-paste. We should be able to have a single definition maven-antrun-plugin and set-included-buidlvers execution under pluginManagement. In that definition `<arg value="${dist_buildvers}"/>`. In the profiles we then only need to define the right values for dist_buildvers such as `snap_and_no_snap_with_databricks`

Reply:
I have addressed this. PTAL